netket/graph/special.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2021 The NetKet Authors - All rights reserved.
```

Comment:
For they are special cases of `Lattice`. But `common_graphs.py` sounds better

Reply:
It not really user-facing API since we provide (and document) the imports from `netket.graph`. But yes, I do prefer `commonm_graphs` (and having descriptive names is useful for us internally as well, of course).